certificate-policy.md

```diff
@@ -12,7 +12,7 @@
 
 **Version 0.0.1**
 
-**October 28, 2016**
+**November 16, 2016**
```

Comment:
This is the line where I'm proposing the date change. I think it's a terrific change.